src/lists.cpp

```diff
@@ -202,6 +202,21 @@ namespace {
 
 // ============================================================================================================
 
+namespace
+{
+   std::map<std::string_view, std::pair<std::string_view, std::string_view>, std::less<>> substitutions {
```

Comment:
Why not `unordered_map`?

Reply:
:shrug: performance doesn't really matter here, and map was less typing! I can change it though.

Reply:
I don't care much, I just saw `map` and noticed we don't need ordering.